src/timed.rs

```diff
@@ -195,14 +199,11 @@ impl<M: Send + 'static, A: Actor<Context = TimedContext<M>, Message = M>> Actor
 
     fn priority(message: &Self::Message) -> Priority {
         match message {
-            // Use underlying message priority if we can reference it.
-            TimedMessage::Instant { message } | TimedMessage::Delayed { message, .. } => {
-                A::priority(message)
-            },
-            // Recurring message is only received once, the recurring instances go through the
-            // internal queue (and not actor's channel). Assign high priority to the request to
-            // set-up the recurrent sending.
-            TimedMessage::Recurring { .. } => Priority::High,
+            // Use underlying message priority for instant messages.
+            TimedMessage::Instant { message } => A::priority(message),
+            // Recurring and Delayed messages are only added to the queue when handled, and then go
+            // through actors priority inboxes again when actually enqueued.
```

Comment:
I'm having trouble visualizing the message flow based on this comment.

> only added to the queue when handled

In my mind, "handled" means the actor has run the `handle()` function on the message, so it's already out of the queue.

Is it possible to be more specific in the language here about the flow of the message from when `send_delayed` is called to when it actually ends up in the actor `handle()` function?

Reply:
Agree. What about

```suggestion
            // These priorities apply to the set-up of Delayed and Recurring messages and when want to handle that pronto.
            // The resulting inner message then comes back as `Instant` and is prioritized per its internal priority.
```

Reply:
@goodhoko you suggestion reads very good to me using it (with very small tweaks)